perf/perfmon.py

```diff
@@ -45,7 +45,7 @@ def setUp(self):
                 self.cancel(
                     "Fail to install %s required for this test." % package)
 
-        git.get_repo('https://git.code.sf.net/p/perfmon2/libpfm4 perfmon2-libpfm4',
+        git.get_repo('git://git.code.sf.net/p/perfmon2/libpfm4',
```

Comment:
Can you please just have it like this 

```
git.get_repo('https://git.code.sf.net/p/perfmon2/libpfm4')
```

Reply:
```
[stdlog] 2024-06-10 06:33:27,840 avocado.utils.process process          L0472 DEBUG| [stdout] Initialized empty Git repository in testsresults/job-2024-06-10T06.33-7be44ae/test-results/1-perfmon.py_Perfmon.test/tmp_dir8leyh85j/1-perfmon.py_Perfmon.test/.git/
[stdlog] 2024-06-10 06:33:27,840 avocado.utils.process process          L0712 INFO | Command '/bin/git init' finished with 0 after 0.002992942s
[stdlog] 2024-06-10 06:33:27,840 avocado.utils.git git              L0129 INFO | Fetching git [REP 'https://git.code.sf.net/p/perfmon2/libpfm4 perfmon2-libpfm4' BRANCH 'master'] -> testsresults/job-2024-06-10T06.33-7be44ae/test-results/1-perfmon.py_Perfmon.test/tmp_dir8leyh85j/1-perfmon.py_Perfmon.test
[stdlog] 2024-06-10 06:33:27,840 avocado.utils.process process          L0655 INFO | Running '/bin/git fetch -q -f -u -t https://git.code.sf.net/p/perfmon2/libpfm4 perfmon2-libpfm4 master:master'
[stdlog] 2024-06-10 06:33:28,045 avocado.utils.process process          L0472 DEBUG| [stderr] fatal: couldn't find remote ref perfmon2-libpfm4
[stdlog] 2024-06-10 06:33:28,046 avocado.utils.process process          L0712 INFO | Command '/bin/git fetch -q -f -u -t https://git.code.sf.net/p/perfmon2/libpfm4 perfmon2-libpfm4 master:master' finished with 128 after 0.205326426s
[stdlog] 2024-06-10 06:33:28,048 avocado.test stacktrace       L0041 ERROR|
[stdlog] 2024-06-10 06:33:28,048 avocado.test stacktrace       L0043 ERROR| Reproduced traceback from: /usr/local/lib/python3.10/dist-packages/avocado/core/test.py:592
[stdlog] 2024-06-10 06:33:28,049 avocado.test stacktrace       L0050 ERROR| Traceback (most recent call last):
[stdlog] 2024-06-10 06:33:28,049 avocado.test stacktrace       L0050 ERROR|   File "/tmp/perfmon/avocado-misc-tests/perf/perfmon.py", line 48, in setUp
[stdlog] 2024-06-10 06:33:28,049 avocado.test stacktrace       L0050 ERROR|     git.get_repo('https://git.code.sf.net/p/perfmon2/libpfm4 perfmon2-libpfm4',
[stdlog] 2024-06-10 06:33:28,049 avocado.test stacktrace       L0050 ERROR|   File "/usr/local/lib/python3.10/dist-packages/avocado/utils/git.py", line 240, in get_repo
[stdlog] 2024-06-10 06:33:28,049 avocado.test stacktrace       L0050 ERROR|     repo.execute()
[stdlog] 2024-06-10 06:33:28,049 avocado.test stacktrace       L0050 ERROR|   File "/usr/local/lib/python3.10/dist-packages/avocado/utils/git.py", line 203, in execute
[stdlog] 2024-06-10 06:33:28,049 avocado.test stacktrace       L0050 ERROR|     self.fetch(self.uri)
[stdlog] 2024-06-10 06:33:28,049 avocado.test stacktrace       L0050 ERROR|   File "/usr/local/lib/python3.10/dist-packages/avocado/utils/git.py", line 135, in fetch
[stdlog] 2024-06-10 06:33:28,049 avocado.test stacktrace       L0050 ERROR|     self.git_cmd(f"fetch -q -f -u -t {uri} {self.branch}:{self.lbranch}")
[stdlog] 2024-06-10 06:33:28,049 avocado.test stacktrace       L0050 ERROR|   File "/usr/local/lib/python3.10/dist-packages/avocado/utils/git.py", line 121, in git_cmd
[stdlog] 2024-06-10 06:33:28,049 avocado.test stacktrace       L0050 ERROR|     return process.run(
[stdlog] 2024-06-10 06:33:28,049 avocado.test stacktrace       L0050 ERROR|   File "/usr/local/lib/python3.10/dist-packages/avocado/utils/process.py", line 1010, in run
[stdlog] 2024-06-10 06:33:28,049 avocado.test stacktrace       L0050 ERROR|     raise CmdError(cmd, sp.result)
[stdlog] 2024-06-10 06:33:28,049 avocado.test stacktrace       L0050 ERROR| avocado.utils.process.CmdError: Command '/bin/git fetch -q -f -u -t https://git.code.sf.net/p/perfmon2/libpfm4 perfmon2-libpfm4 master:master' failed.
[stdlog] 2024-06-10 06:33:28,049 avocado.test stacktrace       L0050 ERROR| stdout: b''
[stdlog] 2024-06-10 06:33:28,049 avocado.test stacktrace       L0050 ERROR| stderr: b"fatal: couldn't find remote ref perfmon2-libpfm4\n"
[stdlog] 2024-06-10 06:33:28,049 avocado.test stacktrace       L0050 ERROR| additional_info: None
[stdlog] 2024-06-10 06:33:28,049 avocado.test stacktrace       L0051 ERROR|
[stdlog] 2024-06-10 06:33:28,051 avocado.test test             L0709 ERROR| ERROR 1-perfmon.py:Perfmon.test -> TestSetupFail: Command '/bin/git fetch -q -f -u -t https://git.code.sf.net/p/perfmon2/libpfm4 perfmon2-libpfm4 master:master' failed.
```

@PraveenPenguin  here its not git clone [url]
But instead its git init; git fetch [url];

But ya `git.get_repo('https://git.code.sf.net/p/perfmon2/libpfm4')` works fine as well

will update and send again